tests/test-definition.yaml

```diff
@@ -17,15 +17,16 @@ dimensions:
   - name: hive
     values:
       - 3.1.3
+      - 4.0.0
       # Alternatively, if you want to use a custom image, append a comma and the full image name to the product version
       # as in the example below.
-      # - 3.1.3,docker.stackable.tech/sandbox/hive:3.1.3-stackable0.0.0-dev
+      # - 4.0.0,docker.stackable.tech/sandbox/hive:4.0.0-stackable0.0.0-dev
   - name: hive-latest
     values:
-      - 3.1.3
+      - 4.0.0
```

Comment:
We had this plan with replacing that "latest" thing. I would rather have some tests for the experimental and have all tests run on the LTS version?
So i would argue to put the 3.1.3 here rather 4.0.0 even if it does not make sense semantically?